packages/automerge-repo-network-websocket/src/BrowserWebSocketClientAdapter.ts

```diff
@@ -43,7 +44,6 @@ export class BrowserWebSocketClientAdapter extends WebSocketNetworkAdapter {
       log(`@ ${this.url}: open`)
       clearInterval(this.timerId)
       this.timerId = undefined
-      this.join()
```

Comment:
Interestingly, @pvh opened #187 which removed the other join. Is that the better option? I don't suppose there is an advantage either way.

Reply:
`join` is already idempotent so it doesn't really matter. I think this is a little clearer because there's one less calback involved.